CMakeLists.txt

```diff
@@ -19470,7 +19470,7 @@ generate_pkgconfig(
   "gRPC"
   "high performance general RPC framework"
   "${gRPC_CORE_VERSION}"
-  "gpr"
+  "gpr openssl"
   "-lgrpc -laddress_sorting -lcares -lz"
```

Comment:
@zackgalbreath  it would be nice to determine the list of dependencies based on our actual dependencies. Like this we have to maintain several versions of the same list.

https://github.com/grpc/grpc/blob/b0c87fd26b446afc98b49bbb7756d5d8e45a961f/test/distrib/cpp/run_distrib_test_cmake_pkgconfig.sh#L58